packages/mdc-tab-indicator/_mixins.scss

```diff
@@ -17,47 +17,41 @@
 
 @import "@material/theme/mixins";
 
-@mixin mdc-tab-indicator-active {
-  opacity: 1;
-}
-
-@mixin mdc-tab-indicator-color($color) {
-  @include mdc-tab-indicator-bar-color($color);
-  @include mdc-tab-indicator-icon-color($color);
+@mixin mdc-tab-indicator-surface {
+  display: flex;
 }
 
-@mixin mdc-tab-indicator-height($height) {
-  @include mdc-tab-indicator-bar-height($height);
-  @include mdc-tab-indicator-icon-height($height);
+@mixin mdc-tab-indicator-active_ {
+  opacity: 1;
 }
 
 @mixin mdc-tab-indicator-bar-color($color) {
-  .mdc-tab-indicator__bar {
+  &.mdc-tab-indicator--bar {
```

Comment:
I think you should assume the client calls `mdc-tab-indicator-bar-color` on a tab indicator which is `--bar`. You don't need the `&.mdc-tab-indicator--bar`. In fact it could lead to overly specific CSS.

Same for below

Reply:
Sounds good.

We'll need more specificity for the icon-height mixin, otherwise we're reliant on load order for font-size. Material Icons explicitly sets the font-size to 24px.